setup.py

```diff
@@ -63,6 +63,7 @@ def read(*names):
     entry_points={
         'console_scripts': [
             'howdoi = howdoi.howdoi:command_line_runner',
+            'keep = keep new | keep list'
```

Comment:
Oops. In reviewing my own code I forgot that we weren't going with this route anymore.